tests/python/contrib/test_hexagon/topi/test_conv2d_fp16_intrin.py

```diff
@@ -195,7 +195,7 @@ class TestConv2dIntrin:
     inp_offset = tvm.testing.parameter((0, 0), ids=["offset0x0"])
 
     @tvm.testing.requires_hexagon
-    def test_conv2d(self, act_shape, wgt_shape, inp_stride, inp_offset, hexagon_session):
+    def DISABLED_test_conv2d(self, act_shape, wgt_shape, inp_stride, inp_offset, hexagon_session):
```

Comment:
Disabling the test so the kernel can be debugged on hardware.  This also has been failing in main.

This simpler test case from @supersat might be easier to debug:

  shape_parameters = [
    (
        (1, 8, 4, 3),
        (3, 3, 3, 3),
        (1, 1),
    ),

...

        act = np.full(act_shape, 1, "float16")
        wgt = np.full(wgt_shape, 1, "float16")

Expected:

[[[[4. 4. 4. 4.]
   [4. 4. 4. 4.]
   [4. 4. 4. 4.]
   [4. 4. 4. 4.]]

  [[4. 4. 4. 4.]
   [4. 4. 4. 4.]
   [4. 4. 4. 4.]
   [4. 4. 4. 4.]]

  [[4. 4. 4. 4.]
   [4. 4. 4. 4.]
   [4. 4. 4. 4.]
   [4. 4. 4. 4.]]

  [[4. 4. 4. 4.]
   [4. 4. 4. 4.]
   [4. 4. 4. 4.]
   [4. 4. 4. 4.]]

  [[4. 4. 4. 4.]
   [4. 4. 4. 4.]
   [4. 4. 4. 4.]
   [4. 4. 4. 4.]]

  [[4. 4. 4. 4.]
   [4. 4. 4. 4.]
   [4. 4. 4. 4.]
   [4. 4. 4. 4.]]

  [[4. 4. 4. 4.]
   [4. 4. 4. 4.]
   [4. 4. 4. 4.]
   [4. 4. 4. 4.]]

  [[4. 4. 4. 4.]
   [4. 4. 4. 4.]
   [4. 4. 4. 4.]
   [4. 4. 4. 4.]]]]

Actual:
[[[[-19936.         4.066      3.957      4.02 ]
   [     4.004      4.004      3.363      4.004]
   [     4.004      4.004      4.01    -138.4  ]
   [ 46144.         4.01       4.004      3.951]]

  [[     4.02       4.004      3.988      4.004]
   [     4.004      4.004     86.06    -311.5  ]
   [  -211.2        4.066      4.004  -7012.   ]
   [     6.348      7.707      4.004      4.3  ]]

  [[     4.06     734.5        4.02       4.004]
   [     4.004      4.043      4.004     28.08 ]
   [     3.418      3.99       4.004   1591.   ]
   [     4.004      4.004      0.869      4.02 ]]

  [[    64.2       15.695      3.994      4.004]
   [     4.004      4.004      8.15   -1981.   ]
   [-11880.     -5380.         4.02       4.035]
   [  4876.         4.02       4.004      3.998]]

  [[     4.004      4.004      4.004      4.24 ]
   [     4.207    -24.92       4.004   -135.6  ]
   [     4.004     43.7        3.97       3.697]
   [     4.184      4.004      4.01     -44.6  ]]

  [[     3.855      4.066      4.09      12.414]
   [       nan      4.004      4.004      4.004]
   [     4.004      4.004   9192.       854.5  ]
   [     4.02       3.355     28.58       4.01 ]]

  [[     4.004    721.         4.004      4.004]
   [     3.934  13912.         4.004   -279.2  ]
   [    18.45     -36.56       3.824    113.8  ]
   [     4.027     11.32       4.19       4.004]]

  [[     4.02       3.799     28.6      500.2  ]
   [     7.598      4.035      4.004      3.715]
   [    77.56    -137.2   -37856.     -1439.   ]
   [     4.035      3.988    518.5        4.004]]]]

Reply:
cc:  @quic-sanirudh , @kparzysz-quic